# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1546706489 -3600
#      Sat Jan 05 17:41:29 2019 +0100
# Branch stable
# Node ID a3a0dbd54a77fcee0c6f7ba9c8a448188066b7f8
# Parent  abe3e8597a5505bb018d233278bfad39fdf2660c
Fix static analyzer detected issues in octave-qt-link (bug #55347)

* octave-qt-link.cc (ctor): fix missing initialization of class members

diff --git a/libgui/src/octave-qt-link.cc b/libgui/src/octave-qt-link.cc
--- a/libgui/src/octave-qt-link.cc
+++ b/libgui/src/octave-qt-link.cc
@@ -51,17 +51,17 @@ along with Octave; see the file COPYING.
 #include "resource-manager.h"
 
 Q_DECLARE_METATYPE (octave_value)
 Q_DECLARE_METATYPE (octave::symbol_info_list)
 
 namespace octave
 {
   octave_qt_link::octave_qt_link (void)
-    : octave_link ()
+    : octave_link (), m_shutdown_confirm_result (false)
   {
     qRegisterMetaType<octave_value> ("octave_value");
     qRegisterMetaType<symbol_info_list> ("symbol_info_list");
   }
 
   bool octave_qt_link::do_confirm_shutdown (void)
   {
     // Lock the mutex before emitting signal.
